examples/hdf5_examples/h5_interm_group.patch

```diff
@@ -1,16 +1,16 @@
---- h5_interm_group.c	2022-03-16 18:21:18.837044757 -0500
-+++ h5_interm_group_logvol.c	2022-03-16 18:21:46.457044898 -0500
-@@ -88,10 +88,13 @@
+--- h5_interm_group.c	2024-11-13 11:56:00.913059201 -0600
++++ h5_interm_group_logvol.c	2024-11-13 11:56:54.281932771 -0600
+@@ -84,11 +84,12 @@
+         g2_id  = H5Gopen2(file, "/G1/G2", H5P_DEFAULT);
          status = H5Gget_info(g2_id, &g2_info);
          printf("Group /G1/G2 has %d member(s)\n", (int)g2_info.nlinks);
- 
+-
 +#if 0
```

Comment:
It seems to pass for me if I activate the code.

Reply:
Looking at the logs,

HDF5-DIAG: Error detected in HDF5 (2.0.0) MPI-process 0:
  #000: ../../src/H5L.c line 1514 in H5Lget_name_by_idx(): unable to get link name
    major: Links
    minor: Can't get value
  #001: ../../src/H5VLcallback.c line 5179 in H5VL_link_get(): link get failed
    major: Virtual Object Layer
    minor: Can't get value
  #002: ../../src/H5VLcallback.c line 5147 in H5VL__link_get(): link get failed
    major: Virtual Object Layer
    minor: Can't get value

It passes because error checking is not done in the examples.